### PR TITLE
Language switch

### PR DESCRIPTION
**Pull request by Selevn**
Install css modules, add language toggle option.

### CODE REVIEW

File: src/js/components/App.container.jsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import React, {useState, useEffect} from "react";
+
+import App from "./App.js"
+
+import {LanguageContext} from './Contexts/LanguageContext.js';
+
+function AppContainer(props) {
+  let [language, setLanguage] = useState("en");
+
+  function toggleLanguageFunc() {
```
Comment (wldyslw) [inline src/js/components/App.container.jsx:10]: Avoid using plain function declaration in functional components. Utilize `useCallback` with dependencies instead

File: src/js/components/Main/Home/Home.container.jsx [hunk 2]
```diff
@@ -13,28 +13,31 @@ function HomeContainer(props) {
...
-      return val ?
-          concerts.filter((i) => i.band.toUpperCase().startsWith(val.toUpperCase())) :
-          []
-    }
-  }
 
   function onInputChange(event) {
     setInputText(event.target.value);
     setHelpList(similarConcerts(event.target.value));
   }
```
Comment (Selevn) [inline src/js/components/Main/Home/Home.container.jsx:39]: How to rewrite this function via "useContext"?

File: src/js/components/App.container.jsx (hunk 1 above)
Comment (Selevn) [reply src/js/components/App.container.jsx]: Rewrited. Check question in latest commit, please

File: src/js/components/Main/Home/Home.container.jsx (hunk 2 above)
Comment (wldyslw) [reply src/js/components/Main/Home/Home.container.jsx:35]: Do you mean `useCallback`?

**Comment by Selevn** [comment]
Code was autoformatted. This code will checked via eslint after merge with dev branch where it is installed and correctly setup. Ok?

**Pull request merged**